### PR TITLE
Program type constant updated. Only shows for grantee reports

### PR DESCRIPTION
**Pull request by jasalisbury**
**Description of change**

Program type constant updated to the values defined in the linked ticket. Program type field only shows when the report is for grantees, it is hidden for non-grantees.

**How to test**

1) Pull down
2) Start a report, select `grantee` and note `Program type(s)` is shown
3) Select `non-grantee` and note `Program type(s)` is hidden

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/170

**Checklist**

- [x] Meets issue criteria
- [x] Code tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [x] Documentation updated
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified

### CODE REVIEW

**Comment by dcmcand** [comment]
Program types still shows up on the review page if you select nongrantee (with no value). Is that correct?

Also, would it be better to have it show up when you select grantee rather than go away when you select nongrantee? Currently, you can select a program type then select nongrantee. Your selection isn't saved or anything, I just wonder if it might be confusing.

**Comment by jasalisbury** [comment]
> Program types still shows up on the review page if you select nongrantee (with no value). Is that correct?

I don't see a good way of hiding it right now, something to keep in mind when we refactor the review accordions to support the download links

> Also, would it be better to have it show up when you select grantee rather than go away when you select nongrantee? Currently, you can select a program type then select nongrantee. Your selection isn't saved or anything, I just wonder if it might be confusing.

Good catch, I'll make it hidden until the user selects `grantee`

**Review by dcmcand — APPROVED** [review]
Very nice.

**Review by gopar — APPROVED** [review]

File: frontend/src/pages/ActivityReport/__tests__/index.js [hunk 1]
```diff
@@ -85,6 +85,15 @@ describe('ActivityReport', () => {
     });
   });
 
+  it('hides program type if non-grantee is selected', async () => {
```
Comment (rahearn) [inline frontend/src/pages/ActivityReport/__tests__/index.js:88]: Can we switch the description to match the behavior now - hidden until shown if grantee is selected.

**Review by rahearn — CHANGES_REQUESTED** [review]
Good stuff, just the one cleanup task on the test name.

**Review by rahearn — APPROVED** [review]
🚢

**Pull request merged**